src/Compilers/CSharp/Test/Emit3/Semantics/ExtensionTests.cs

```diff
@@ -19099,6 +19099,810 @@ public void AppendFormatted<T>(T hole, int alignment = 0, string format = null)
             Diagnostic(ErrorCode.ERR_InterpolatedStringHandlerMethodReturnMalformed, "{f2()}").WithArguments("?.()").WithLocation(1, 21));
     }
 
+    [Theory, WorkItem("https://github.com/dotnet/roslyn/issues/78137")]
```

Comment:
Does this PR also by any chance fix https://github.com/dotnet/roslyn/issues/78435?